src/coreclr/vm/frames.h

```diff
@@ -2868,8 +2870,13 @@ class InlinedCallFrame : public Frame
     virtual void UpdateRegDisplay(const PREGDISPLAY);
 
     // m_Datum contains MethodDesc ptr or
-    // - on AMD64: CALLI target address (if lowest bit is set)
-    // - on X86: argument stack size (if value is <64k)
+    // - on 64 bit host: CALLI target address (if lowest bit is set)
```

Comment:
Does https://github.com/dotnet/runtime/blob/main/src/coreclr/vm/amd64/PInvokeStubs.asm#L33-L38 need to be adjusted for the stolen bits?

Reply:
It doesn't need to be updated. When the lowest bit is set, we never fiddle with the newly added bits.

Reply:
I'll update the comment to make it clearer.